src/main/java/pl/put/poznan/json/tools/service/ComparisonService.java

```diff
@@ -8,7 +8,41 @@
 @Service
 public class ComparisonService {
 
-    public List<Integer> compare(String s, String s1) {
-        return new ArrayList<Integer>();
+    private String[] json1;
+    private String[] json2;
+
+    private String trimJson(String json) {
+        return json.trim();
+    }
+
+    private String[] splitJson(String json) {
+        return json.split("\\r?\\n");
+    }
+
+    public List<Integer> compare(String json1, String json2) {
+        json1 = trimJson(json1);
+        json2 = trimJson(json2);
+        if (json1.equals(json2))
```

Comment:
to możesz porównać od razu w ifie i nie takie powinny być atrybuty bo więcej ich nie wykorzystujesz przecież